### PR TITLE
Bugfix: capture in openpulse

### PR DESCRIPTION
**Pull request by lcauser-oqc**
Using `capture` in openpulse will generate an acquire instruction that has a delay `None`, which gives errors when trying to do arithmetic with `NoneType` and `float`

Changes:
- If it exists, use the default delay for a measure acquire.
- Added a redundancy measure to ensure that if the delay is `None`, use `0.0` instead.
- Test to ensure it takes the default value.

### CODE REVIEW

File: src/qat/purr/integrations/qasm.py [hunk 1]
```diff
@@ -1995,12 +2001,36 @@ def _capture_iq_value(
         # these post processing operations are removed for executions on live hardware.
         #
         # The returned value for each shot after postprocessing is a complex iq value.
+
+        # Determine the delay for the channel
+        delay = 0.0
+        if pulse_channel.channel_type == ChannelType.acquire:
+            # TODO: remove full id when previous PR is merged
```
Comment (bgsach) [inline src/qat/purr/integrations/qasm.py:2008]: Is this TODO something you can action now?

Comment (lcauser-oqc) [reply src/qat/purr/integrations/qasm.py:2008]: Will action in #202 :smile:

File: src/qat/purr/backends/qblox/live.py [hunk 2]
```diff
@@ -69,7 +69,9 @@ def _execute_on_hardware(
                 for aq in aqs:
                     physical_channel = aq.physical_channel
                     dt = physical_channel.sample_time
-                    physical_channel.readout_start = aq.start * dt + aq.delay
+                    physical_channel.readout_start = aq.start * dt + (
+                        aq.delay if aq.delay else 0.0
```
Comment (keriksson-rosenqvist) [inline src/qat/purr/backends/qblox/live.py:73]: Minor but I would prefer more explicit if checks.

**Review by keriksson-rosenqvist — APPROVED** [review]

File: src/qat/purr/backends/qblox/live.py (hunk 2 above)
Comment (bgsach) [reply src/qat/purr/backends/qblox/live.py:73]: like `aq.delay or 0.0` ?

Comment (bgsach) [reply src/qat/purr/backends/qblox/live.py:73]: (the last comment was a joke in case that wasn't clear)

Comment (keriksson-rosenqvist) [reply src/qat/purr/backends/qblox/live.py:73]: :joy:

**Pull request merged**